javascript/elements/futurism_utils.js

```diff
@@ -25,6 +25,9 @@ const observerCallback = (entries, observer) => {
 }
 
 export const extendElementWithIntersectionObserver = element => {
+  // IntersectionObserver takes an options object as a 2nd argument
+  // https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API
+  // Set rootMargin to make content load right before you reach it
```

Comment:
all good, but how is the user supposed to set that property? I think we'd need to define attributes on the `<futurism-... rootMargin="" ...>` element and use them here?

btw, there's also `threshold` ...

Reply:
This made sense when the file was originally ejected...